db/migrate/20190625173111_create_icons.rb

```diff
@@ -0,0 +1,16 @@
+class CreateIcons < ActiveRecord::Migration[5.2]
+  def change
+    create_table :icons do |t|
+      t.string :data
+      t.string :source_ref
+      t.string :source_id
+      t.bigint :portfolio_item_id
+      t.bigint :tenant_id
+      t.index :tenant_id
+
+      t.timestamps
+    end
+
+    remove_column :portfolio_items, :service_offering_icon_ref, :string
```

Comment:
Should we include a migration to collect icons for any previously stored icons?

Reply:
Good catch, I'd have nuked every icon in the current database if we ran this :| 
Added.